gadgets/trace_mount/go/program.go

```diff
@@ -99,9 +99,9 @@ func gadgetInit() int {
 		dest, _ := destField.String(data)
 		fs, _ := fsField.String(data)
 		dataStr, _ := dataField.String(data)
-		ret, _ := retField.Int32(data)
+		retRaw, _ := retRawField.Int32(data)
 
-		callField.SetString(data, getCallStr(opRaw, src, dest, fs, flags, dataStr, ret))
+		callField.SetString(data, getCallStr(opRaw, src, dest, fs, flags, dataStr, retRaw))
```

Comment:
Why not to provide the human readable version here too?

Reply:
I think the priority of the operators is getting in the way. I tried to get the `ret` field, but it is apparently not known